Library/Formula/ruby.rb

```diff
@@ -56,7 +58,7 @@ def install
       args << "--with-arch=#{Hardware::CPU.universal_archs.join(",")}"
     end
 
-    args << "--program-suffix=23" if build.with? "suffix"
+    args << "--program-suffix=#{program_suffix}" if build.with? "suffix"
```

Comment:
Is the `if build.with? "suffix"` still necessary here given the `def program_suffix` is wrapped in its own `build.with?` conditional?